### DISCUSSION TITLE
Library free search on Giardia dB: library from DIA but not from fasta?

### DISCUSSION THREAD

**Discussion by serenacamerini73** [vdemichev/DiaNN#845] [Q&A]
We performed data analysis on Giardia samples in library free mode adding Giardia dB .fasta downloaded from https://giardiadb.org/giardiadb/app.
Strangely, in the in the report log file there were these annotations:
Loading FASTA C:\DataBase\GiardiaDB-35_GintestinalisAssemblageAWB_AnnotatedProteins_corrected.fasta
[0:01] Loading FASTA C:\DataBase\GiardiaDB-65_GintestinalisAssemblageAWB2019_AnnotatedProteins.fasta
[0:03] Processing FASTA
[0:11] Assembling elution groups
[0:18] 2806018 precursors generated
[0:18] Gene names missing for some isoforms
[0:18] Li**brary contains 1 proteins, and 1 genes**

Then it was reported
Initialising library

[20:24] First pass: generating a spectral library from DIA data...
....and so on

It seems that the only spectral library (except  fror just 1 protein) was generated from DIA data.

Nevertheless, we obtained that  more than 4000  proteins were quantified and described with the relative accession number, but not the protein description .

What was wgrong, if something?

Thank you a lot

Serena

**Comment by vdemichev** [comment]
Hi Serena,

If you look at the main report of DIA-NN, the sequence IDs and protein groups there should be correct. What's not being extracted is UniProt names & genes from the FASTA, but this is expected as I guess this is not a UniProt FASTA. You can always add annotations to the main report easily with some R package that loads FASTAs.

Best,
Vadim

**Comment by serenacamerini73** [comment]
Clear all the answers....hhank you a lot!

Serena

________________________________
Da: Vadim Demichev ***@***.***>
Inviato: martedì 31 ottobre 2023 10:56
A: vdemichev/DiaNN ***@***.***>
Cc: Camerini Serena ***@***.***>; Author ***@***.***>
Oggetto: Re: [vdemichev/DiaNN] Library free search on Giardia dB: library from DIA but not from fasta? (Discussion #845)


Hi Serena,

If you look at the main report of DIA-NN, the sequence IDs and protein groups there should be correct. What's not being extracted is UniProt names & genes from the FASTA, but this is expected as I guess this is not a UniProt FASTA. You can always add annotations to the main report easily with some R package that loads FASTAs.

Best,
Vadim

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/vdemichev/DiaNN/discussions/845*discussioncomment-7433337__;Iw!!OmMtAkQYH20LOJIZAzSBroM!5P5jZz1TZZszq5TLc8Cpe73IZfH-mscqxikUucbKO0gUrYXNP6qhe9qrCtKLeXfvuf9ZQgxAAX5byDABXTGt428QSBU$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/BBLFJ4HR7XIJXSXLKOPQLILYCDDOJAVCNFSM6AAAAAA6PTGSPKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMZTGMZTO__;!!OmMtAkQYH20LOJIZAzSBroM!5P5jZz1TZZszq5TLc8Cpe73IZfH-mscqxikUucbKO0gUrYXNP6qhe9qrCtKLeXfvuf9ZQgxAAX5byDABXTGtdREr5go$>.
You are receiving this because you authored the thread.



[https://www.iss.it/o/iss/images/ISS.png]
Scegli l'Istituto Superiore di Sanità. Dona il 5x1000 alla Ricerca Sanitaria
Firma e inserisci il Codice Fiscale 80211730587
________________________________
Per saperne di più clicca qui<[https://www.iss.it/-/l-iss-sostiene-i-giovani-ricercatori-con-il-5x1000-1]>
